libbpf-rs/src/program.rs

```diff
@@ -866,6 +896,36 @@ impl Program {
         })
     }
 
+    /// Dry-run the program with the given input data.
+    ///
+    /// This function uses the
+    /// [BPF_PROG_RUN](https://www.kernel.org/doc/html/latest/bpf/bpf_prog_run.html)
+    /// facility.
+    pub fn invoke(&mut self, input: Input<'_>) -> Result<Output> {
```

Comment:
I think it might be hard for people to find invoke() or realize that it is implementation of `bpf_prog_test_run()`. So I'd call it either `run()` or `test_run()`, just to stay close to kernel/libbpf terminology and aid in discovering it.